src/guiguts/widgets.py

```diff
@@ -65,7 +66,11 @@ def __init__(
 
         self._do_config()
         self.save_config = False
-        self.bind("<Configure>", self._handle_config)
+        self.bind("<Configure>", self._handle_config)  #
```

Comment:
Why the trailing `#` here? Seems an unnecessary change...

Reply:
Oops - thanks